lib/reporters/spec.js

```diff
@@ -67,20 +67,34 @@ function Spec(runner, options) {
   });
 
   runner.on(EVENT_TEST_PASS, function (test) {
+    function logRetries() {
+      if (test._currentRetry > 0) {
```

Comment:
The underscore in `_currentRetry` is an old-school practice for indicating that its a private property, so we shouldn't rely on it here I think as its not intended for outside consumption.

~~If we want it to be exposed we should make it an explicit public property, like `duration`, and probably name it `retries` or `retryCount` or such~~

We should use the public [`.currentRetry()`](https://github.com/mochajs/mocha/blob/24560c1532b8e67d1254b489fcc6f84c4033c0e1/lib/runnable.js#L185-L195) instead.